subprojects/core/src/main/java/org/gradle/api/internal/catalog/problems/VersionCatalogProblemId.java

```diff
@@ -23,21 +23,31 @@
  */
 public enum VersionCatalogProblemId {
 
-    ACCESSOR_NAME_CLASH,
-    CATALOG_FILE_DOES_NOT_EXIST,
-    INVALID_ALIAS_NOTATION,
-    INVALID_DEPENDENCY_NOTATION,
-    INVALID_PLUGIN_NOTATION,
-    INVALID_MODULE_NOTATION,
-    TOO_MANY_IMPORT_FILES,
-    NO_IMPORT_FILES,
-    TOO_MANY_IMPORT_INVOCATION,
-    RESERVED_ALIAS_NAME,
-    TOML_SYNTAX_ERROR,
-    TOO_MANY_ENTRIES,
-    UNDEFINED_ALIAS_REFERENCE,
-    UNDEFINED_VERSION_REFERENCE,
-    UNSUPPORTED_FILE_FORMAT,
-    UNSUPPORTED_FORMAT_VERSION,
-    ALIAS_NOT_FINISHED
+    ACCESSOR_NAME_CLASH("Accessor name clash"),
+    CATALOG_FILE_DOES_NOT_EXIST("Import of external catalog file failed"),
+    INVALID_ALIAS_NOTATION("The alias is invalid"),
+    INVALID_DEPENDENCY_NOTATION("The dependency notation is not valid"),
+    INVALID_PLUGIN_NOTATION("The plugin notation is not valid"),
+    INVALID_MODULE_NOTATION("The module notation is not valid"),
+    TOO_MANY_IMPORT_FILES("Importing multiple files is not supported"),
+    NO_IMPORT_FILES("No files were resolved to be imported"),
+    TOO_MANY_IMPORT_INVOCATION("The 'from' method can only be called once"),
+    RESERVED_ALIAS_NAME("The alias is not valid"),
```

Comment:
```suggestion
    RESERVED_ALIAS_NAME("The alias is invalid"),
```
just diffing the msg i could not tell any difference to `INVALID_ALIAS_NOTATION`, which one is what.

```suggestion
    RESERVED_ALIAS_NAME("The alias is reserved"),
```